migrations/20201109030750-add-user_id-to-organization-application.ts

```diff
@@ -0,0 +1,40 @@
+import Base from 'db-migrate-base';
+
+/**
+ * Add foreign key to Organization Applications table for many to 1 relationship with Users
```

Comment:
Was there a specific reason for connecting the many to one relationship to Users instead of Organizations?